moto/resourcegroupstaggingapi/models.py

```diff
@@ -131,20 +136,31 @@ def tag_filter(tag_list: List[Dict[str, str]]) -> bool:
             else:
                 return True
 
-        def format_tags(tags: Dict[str, str]) -> List[Dict[str, str]]:
+        def format_tags(
+            tags: Dict[str, Optional[str]]
+        ) -> List[Dict[str, Optional[str]]]:
             result = []
             for key, value in tags.items():
                 result.append({"Key": key, "Value": value})
             return result
 
         def format_tag_keys(
-            tags: List[Dict[str, str]], keys: List[str]
-        ) -> List[Dict[str, str]]:
+            tags: List[Dict[str, Optional[str]]], keys: List[str]
+        ) -> List[Dict[str, Optional[str]]]:
             result = []
             for tag in tags:
                 result.append({"Key": tag[keys[0]], "Value": tag[keys[1]]})
             return result
 
+        # ACM
+        if not resource_type_filters or "acm" in resource_type_filters:
+            for certificate in self.acm_backend._certificates.values():
+                tags = format_tags(certificate.tags)
```

Comment:
Can you try to rename this variable `tags` to something service-specific, maybe `acm_tags`? I wonder if that fixes the linting problems.

We're essentially reusing the `tags` variable for all services, so I wonder if the problem is just that `mypy` is unhappy with the fact we're re-assigning the variable to different types.

Reply:
I am on mypy `v.1.5.1`. I changed the variable to `acm_tags` but still getting the same errors.

Reply:
Hope you don't mind - I just pushed a 'fix' by just changing it to `Dict[str, Any]`. I had already made the change anyway, so there was no point in undoing all that, and then asking you to make the same changes..

It does feel like a `mypy` bug/shortcoming, that it doesn't recognize that the two should be interchangeable.

Resorting to `Any` is not pretty, but at least it works. I've left a note in there (mostly to myself) to replace it with TypedDict later on, that should fix it completely.

Reply:
Great! Thank you - I don't mind at all. The linting check is passing now, just failing on the testing check now. 
My unit test here is passing:
`tests/test_resourcegroupstaggingapi/test_resourcegroupstaggingapi.py::test_get_resources_acm PASSED`

Reply:
I've restarted the build, and it passed now. It's still marked as draft, I don't know if you wanted to add anything else? As far as I'm concerned this is good to go!

Reply:
Yep, nothing else to add. Gonna change it to ready for review - thanks for your help!